src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import java.util.function.BooleanSupplier;
+import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
+import com.chopshop166.chopshoplib.motors.Modifier;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+import com.chopshop166.chopshoplib.states.SpinDirection;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.maps.RobotMap.IntakeMap;
+
+public class Intake extends SmartSubsystemBase {
+
+    private final SmartMotorController deploymentMotor;
+    private final SmartMotorController rollerMotor;
+
+    private final BooleanSupplier insideLimit;
+    private final BooleanSupplier outsideLimit;
+
+    private final double rollerSpeed = 1;
+    private final double deploySpeed = 1;
```

Comment:
These should be `private final static double` and the naming should be in upper snake case.

See https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names for more info.

```suggestion
    private static final double ROLLER_SPEED= 1;
    private static final double DEPLOY_SPEED= 1;
```